pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -175,6 +199,12 @@ public NamespaceService(PulsarService pulsar) {
         this.bundleSplitListeners = new CopyOnWriteArrayList<>();
         this.localBrokerDataCache = pulsar.getLocalMetadataStore().getMetadataCache(LocalBrokerData.class);
         this.redirectManager = new RedirectManager(pulsar);
+
+        this.lookupLatencyHistogram = pulsar.getOpenTelemetry().getMeter()
+                .histogramBuilder("pulsar.broker.lookup.request.duration")
+                .setDescription("Lookup request duration")
+                .setUnit("s")
```

Comment:
Is there a specific reason to use second as a unit for latency? 
I'd assume that microseconds or nanoseconds would be more useful and avoid unnecessary floating point number operations.

Reply:
As the the whole world will use those conventions, we better follow them to align.

Reply:
Thank you @dragosvictor for the pointer. I also found https://github.com/open-telemetry/opentelemetry-specification/issues/2977 which was helpful as background information.